test-adapter/src/main/scala/org/scalajs/sbttestadapter/TaskDefSerializers.scala

```diff
@@ -16,7 +16,7 @@ import org.scalajs.core.tools.json._
 import FingerprintSerializers._
 import SelectorSerializers._
 
-object TaskDefSerializers {
+private[scalajs] object TaskDefSerializers {
```

Comment:
Or we don't care and just say breaking change. After all, we don't have very strict rules about the compatibility of the test-adapter.